src/models/simulation.ts

```diff
@@ -53,6 +62,10 @@ export class SimulationModel {
   // specific moments and usually for all the cells, so this approach can be way more efficient.
   @observable public cellsStateFlag = 0;
   @observable public cellsElevationFlag = 0;
+  @observable public cellsBaseStateFlag = 0;
+  @observable public cellsSimulationStateFlag = 0;
```

Comment:
What's the purpose of `cellsBaseStateFlag` and `cellsSimulationStateFlag`?
I can see they're added but not used anywhere in the rendering code, so I guess we don't need them.